backend/src/test/java/kr/touroot/travelogue/service/TravelogueFacadeServiceTest.java

```diff
@@ -419,4 +423,51 @@ void unlikeTravelogueWithNotExist() {
                 .isInstanceOf(BadRequestException.class)
                 .hasMessage("존재하지 않는 여행기입니다.");
     }
+
+
+    @DisplayName("여행기의 좋아요 수가 바뀔 때 해당 여행기가 캐싱되는 기준 이상의 좋아요 수를 가진다면 해당 캐시를 무효화 한다")
+    @Test
+    void cacheEviction() {
+        // given
+        Member liker = testHelper.initKakaoMemberTestData();
+        Travelogue travelogue = testHelper.initTravelogueTestDataWithLike(liker);
+        redisTemplate.opsForValue().set("traveloguePage::1", "cached");
+        // when
+        service.likeTravelogue(travelogue.getId(), new MemberAuth(liker.getId()));
```

Comment:
```suggestion
        redisTemplate.opsForValue().set("traveloguePage::1", "cached");
        
        // when
        service.likeTravelogue(travelogue.getId(), new MemberAuth(liker.getId()));
```
띄어쓰기가 묘하게 됐군요.. 바쁘시지 않다면 수정해주시죠!